{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings/base.py

```diff
@@ -101,18 +101,27 @@
 
 TEMPLATES = [
     {
-        "BACKEND": "django_jinja.backend.Jinja2",
-        "APP_DIRS": True,
-        "OPTIONS": {
-            "match_regex": r"^(?!admin/.*)",
-            "match_extension": '.html',
-            "newstyle_gettext": True,
+        'BACKEND': 'django_jinja.backend.Jinja2',
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'match_regex': r'^(?!!(admin|registration)/.*)',
+            'match_extension': '.html',
```

Comment:
The more I think about it, the more I wonder if maybe having Jinja templates ending in `.html` instead of something like `.jinja` is a bad idea. For every other custom format we have a new file extension, and it seems common in other proejcts to use `.jinja` for Jinja templates.

Thoughts?